### ISSUE TITLE
Platypus in disabled in homebrew

### ISSUE THREAD

**Issue by abelozerov** [sveinbjornt/Platypus#262]
Not able to install platypus from homebrew anymore:

```
brew install platypus --formula
Error: platypus has been disabled because it is not maintained upstream!
```

**Comment by sveinbjornt** [comment]
I've been meaning to look into this. I am not the maintainer of the brew package spec.

**Comment by huyz** [comment]
@sveinbjornt Is it fair to say that it's still maintained though? Looking at your commit history, it sure looks like it

**Comment by sveinbjornt** [comment]
Yes, still maintained. I'm hoping to get around to publishing an updated version soon.

**Comment by huyz** [comment]
See updated note https://github.com/sveinbjornt/Platypus/issues/289#issuecomment-3602563480
